### PR TITLE
updated the wide mobile view

### LINKED ISSUE #3414: [Docs] Mobile view too wide
#### Current State
Meshery Docs viewed on a mobile phone does not properly fit the screen. 

#### Screenshot

FA7A87F7-BBCC-4867-A7BD-8F802C870DDC

---

#### Contributor Resources

- [Meshery documentation site](https://docs.meshery.io/)
- [Meshery documentation source](https://github.com/meshery/meshery/tree/master/docs)
- [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
- [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by warunicorn19**
Signed-off-by: Aditya Chaterjee <speak2adi@gmail.com>

**Description**
wide mobile view in meshery docs fixed

This PR fixes #3414

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 09e52985efacf631304fdc1bcdc74127e4206453 at: https://60ea90db6f32a884848c067c--meshery-docs-preview.netlify.app

**Comment by adithyaakrishna** [comment]
@warunicorn19 There's a small issue regarding the last image at, https://60ea90db6f32a884848c067c--meshery-docs-preview.netlify.app/installation/quick-start

The last image tends to get cut in the middle while other images are shown perfectly

image


@warunicorn19 Do you mind fixing this up in this PR? 👀

**Comment by adithyaakrishna** [comment]
Here's a Screenshot from my mobile 📱

Screenshot_20210711-201609

**Comment by warunicorn19** [comment]
Okay got it, no problem, I'll do it by tomorrow

**Comment by adithyaakrishna** [comment]
Okay sure 🚀

**Review by adithyaakrishna — CHANGES_REQUESTED** [review]
@warunicorn19 All Other Changes LGTM 🎉 

Please lmk once the above change is done, will re-review again 💯

**Comment by warunicorn19** [comment]
Cool😎

**Comment by warunicorn19** [comment]
hey @adithyaakrishna, could you please check if it looks fine now?

**Comment by l5io** [comment]
🚀 Preview for commit 66ffe7ee29381eefbda3a99e70a9790ed09a3e66 at: https://60ebd3fa41189bccb8525852--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit fca0d5f6653c7ad1bd36a475401869c9d33ae470 at: https://60ebdcb93d6d3f8e9a570019--meshery-docs-preview.netlify.app

File: docs/pages/installation/index.md [hunk 1]
```diff
@@ -63,14 +63,14 @@ If your config has not been auto-detected, you may manually locate and upload yo
 Run connectivity tests and verify the health of your Meshery system. Verify Meshery's connection to Kubernetes by clicking on your configuration `context` name. You will be notified of your connection status. You can also verify a successful connection between Meshery and its adapters by clicking on any of the available [Adapter Ports]({{ site.baseurl }}/concepts/architecture#adapter-ports).
 
 <a href="/assets/img/adapters/meshery-ui.png">
-<img style="width:450px;height=auto;" src="/assets/img/adapters/meshery-ui.png" />
+<img style="width:550px;height=auto;" src="/assets/img/adapters/meshery-ui.png" />
```
Comment (leecalcote) [inline docs/pages/installation/index.md:66]: The static sizing concerns me a that responsiveness might be a bit fragile.

If there's a more clever approach that is simple to implement, I encourage us to take moment to consider it. If not, let's press forward.

Comment (warunicorn19) [reply docs/pages/installation/index.md:66]: This is just to take a default value when the page is opened, i have added the responsiveness in the css file. That is the last commit @leecalcote

Comment (leecalcote) [reply docs/pages/installation/index.md:66]: Thanks @warunicorn19 👍

**Review by leecalcote — APPROVED** [review]

**Pull request merged**